Topics/Development_Process/Duckdns.md

```diff
@@ -0,0 +1,53 @@
+# Overcoming a Dynamic IP Address with DuckDNS
+
+## Motivation
+
+So you have chosen to deploy your application on AWS or perhaps you have turned a spare computer of yours into your home server. You are finally deployed and users can finally access your site! I mean, the address for your site may be something weird like `http://123.0.0.321` but at least _someone_ can access your site right?
+
+Well, if you ever have to restart your server to make updates or save some compute power when nobody is using your server, you may find that the next time you boot the server up, the IP address of your deployment has changed! Now you have to tell everyone your application has a new URL. This is because most hosting options (especially the free/low cost options) give your production server what is called a _Dynamic IP address_. In other words, services like AWS or your Internet Service Provider assign a new public (or external) IP address to your server whenever it boots up and joins the network. This is often done to save resources as an IP not used by one person can be used by another.
```

Comment:
```suggestion
Well, if you ever have to restart your server to make updates or save some computing power when nobody is using your server, you may find that the next time you boot the server up, the IP address of your deployment has changed! Now you have to tell everyone your application has a new URL. This is because most hosting options (especially the free/low-cost options) give your production server what is called a _Dynamic IP address_. In other words, services like AWS or your Internet Service Provider (ISP) assign a new public (or external) IP address to your server whenever it boots up and joins the network. This is often done to save resources as an IP not used by one person can be used by another.
```
I added ISP in brackets so that it is more clear when you mention ISP later on.